feature/tabsflow/moreflow/faq/src/main/java/com/epmedu/animeal/tabs/more/faq/util/LoremIpsumGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package com.epmedu.animeal.tabs.more.faq.util
+
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import kotlin.random.Random
+
+/**
+ * Generates Lorem Ipsum text with random length in specified [wordsCountRange]
+ * @param wordsCountRange Words count range for randomizer. By default 5..100
+ */
+internal fun generateLoremIpsum(wordsCountRange: IntRange = 5..100) =
```

Comment:
May be we can move this fun to extensions or some base packages, can be useful across the project

Reply:
Moved to foundation module, because `LoremIpsum` class is from compose library